images.CI/azure-pipelines/image-generation.yml

```diff
@@ -55,6 +55,14 @@ jobs:
                         -ImageName ${{ parameters.image_type }} `
                         -AccessToken $(RELEASE_TARGET_TOKEN)
 
+  - task: PowerShell@2
+    displayName: 'Output Readme file content'
+    inputs:
+      targetType: 'inline'
+      script: |
+        $docsPath = Get-ChildItem -Path "images" -Include ${{ parameters.image_readme_name }} -Recurse -Depth 1 | Select-Object -First 1
+        Get-Content -Path $docsPath
```

Comment:
Also, I believe, such things should be downloaded into build artifacts.

Reply:
If we upload it to build artifacts, we have to download it locally to read and validate. Step logs are available in web UI. If you see that it makes sense, we can publish build artifact in additional to this step. But artifact is not convenient way for initial purpose of PR